cgmanifests/generate_cgmanifest.py

```diff
@@ -72,7 +72,7 @@ def normalize_path_separators(path):
         "foreach",
         "--quiet",
         "--recursive",
-        "{} {} $toplevel/$sm_path".format(
+        "'{}' '{}' $toplevel/$sm_path".format(
```

Comment:
Needed on windows when paths have spaces in them.

Reply:
Thanks. I never got it work on Windows.